substrate-processor/src/batch.ts

```diff
@@ -18,13 +27,25 @@ export interface DataHandlers {
     evmLogs: Record<EvmContractAddress, {filter?: EvmTopicSet[], handler: EvmLogHandler}[]>
 }
 
-
+/**
+ * Defines a batch of blocks to be processed
+ * 
+ * @property range: a {@link Range} defining start and end blocks for the batch
+ * @property handlers: a {@link DataHandlers} objects to collect the handlers that should be triggered for the batch.
```

Comment:
it should be stressed that each handler in a batch is admissible in the given range.